app.css

```diff
@@ -158,3 +159,56 @@ a{
     height: 10rem;
     border-radius: 100%;
 }
+
+/* alerts */
+.alert-div{
```

Comment:
make different CSS file for every component rather than combining it with app.css

Reply:
Yes done!